CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (<https://github.com/openvinotoolkit/datumaro/pull/1005>)
 - Add Correct transformation
   (<https://github.com/openvinotoolkit/datumaro/pull/1006>)
+- Implement AnnotationReindex transform
```

Comment:
could you update the name of transform here too?

Reply:
https://github.com/openvinotoolkit/datumaro/pull/1008/commits/e4977d23a1f7248497bc49c082acf02ad789ab62 fixes it.